test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -642,50 +642,35 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
     # patch Returns PATCH data.
     # put Returns PUT data.
     # delete Returns DELETE data
-    $testMethods = @("GET", "POST", "PATCH", "PUT", "DELETE")
+    $testMethods = @("POST", "PATCH", "PUT", "DELETE")
     $contentTypes = @("text/plain", "application/xml", "application/json")
 
     foreach ($contentType in $contentTypes)
     {
         foreach ($method in $testMethods)
         {
             # Operation options
-            $operation = $method.ToLower()
-            $uri = "http://httpbin.org/$operation"
+            $uri = Get-WebListenerUrl -Test $method
             $body = GetTestData -contentType $contentType
+            $command = "Invoke-WebRequest -Uri $uri -Body '$body' -Method $method -ContentType $contentType"
 
-            if ($method -eq "GET")
-            {
-                $command = "Invoke-WebRequest -Uri $uri"
-            }
-            else
-            {
-                $command = "Invoke-WebRequest -Uri $uri -Body '$body' -Method $method -ContentType $contentType -TimeoutSec 5"
-            }
-
-            It "$command" {
+            It "Invoke-WebRequest -Uri $uri  -Method $method -ContentType $contentType -Body [body data]" {
 
                 $result = ExecuteWebCommand -command $command
                 ValidateResponse -response $result
 
                 # Validate response content
                 $jsonContent = $result.Output.Content | ConvertFrom-Json
                 $jsonContent.url | Should Match $uri
-
-                # For a GET request, there is no data property to validate.
-                if ($method -ne "GET")
+                $jsonContent.headers.'Content-Type' | Should Match $contentType
+                # Validate that the response Content.data field is the same as what we sent.
+                if ($contentType -eq "application/xml")
+                {
+                    $jsonContent.data | Should Be $body
+                }
+                else
                 {
-                    $jsonContent.headers.'Content-Type' | Should Match $contentType
-
-                    # Validate that the response Content.data field is the same as what we sent.
-                    if ($contentType -eq "application/xml")
-                    {
-                        $jsonContent.data | Should Be $body
-                    }
-                    else
-                    {
-                        $jsonContent.data | Should Match $body
-                    }
+                    $jsonContent.data | Should Match $body
```

Comment:
Maybe 'Should Be' too?

Reply:
I can try changing it. I assumed there was specific logic to use match for some good reason. if you look at the if block above they ha `Should Be` for `application/xml` but match for everything else. *shrugs

Reply:
My question was just about the "some good reason". 
Right question is - should we expect that Invoke-WebRequest  change the body (add/remove whitespaces)?

Reply:
fixed